src/control/math_tool.rs

```diff
@@ -165,7 +159,7 @@ pub fn calc_hp_comp_matrix(
     // Tangential hardpoints
     let mut hd_comp_tangent: SMatrix<f64, NUM_ACTIVE_ACTUATOR_TANGENT, NUM_HARDPOINTS_TANGENT> =
         SMatrix::repeat(2.0 / 3.0);
-    if hardpoints_tangent == &vec![72, 74, 76] {
+    if hardpoints_tangent == vec![72, 74, 76] {
```

Comment:
```suggestion
    if hardpoints_tangent == [72, 74, 76] {
```
Correct as written, but my proposed change has the same effect and avoids allocating a new vector. I think. I'm still new to Rust.

Reply:
Improved in the commit: [bfd1315](https://github.com/lsst-ts/ts_mtm2_controller/pull/14/commits/bfd1315ff65beab83977066ba8f245cedcd50624). Thanks!